official/docs/golang/current/shipping-refund/refund.go

```diff
@@ -1,7 +1,7 @@
-package example
+package shipping_refund
```

Comment:
```suggestion
package example
```

This should be `example`, if you look at other code examples for Go